src/adapters/MeetingsJSONAdapter.js

```diff
@@ -92,13 +92,27 @@ export default class MeetingsJSONAdapter extends MeetingsAdapter {
       observer.complete();
     });
 
-    // Attach local media stream if meeting `localVideo` property is not `null`.
+    // Attach media stream if meeting any of media stream property is not `null`.
     // We can not attach the MediaStream object in a JSON module, the work needs
     // to be done here.
-    const getMeetingWithLocalMedia$ = getMeeting$.pipe(
+    const getMeetingWithMedias$ = getMeeting$.pipe(
```

Comment:
```suggestion
    const getMeetingWithMedia$ = getMeeting$.pipe(
```

Reply:
this should be plural. since there could be multiple medias [localVideo, remoteVideo, remoteAudio] attached to the meeting.

Reply:
`media` is actually plural, the singular is `medium`

Reply:
🤓